features/steps/store_resource.py

```diff
@@ -176,11 +175,11 @@ def stored_callback(event: ItemStored, uow: UnitOfWork) -> None:
         context.stored_event = event
 
     handlers: dict[Type[Event], list[Callable]] = {
-        PackageSubmitted: [lambda event, uow: receive_package(event, context.uow)],
-        PackageReceived: [lambda event , uow: verify_package(event, context.uow)],
-        PackageVerified: [lambda event, uow: unpack_item(event, context.uow)],
-        ItemUnpacked: [lambda event, uow: store_item(event, context.uow)],
-        ItemStored: [lambda event, uow: stored_callback(event, context.uow)]
+        PackageSubmitted: [lambda event: receive_package(event, context.uow)],
+        PackageReceived: [lambda event: verify_package(event, context.uow)],
+        PackageVerified: [lambda event: unpack_item(event, context.uow)],
+        ItemUnpacked: [lambda event: store_item(event, context.uow)],
+        ItemStored: [lambda event: stored_callback(event, context.uow)]
```

Comment:
`uow` is already injected via the context, so it just ends up unused if we add it to the lambda signature here.